test/c/memory-safety/global_and_heap_alloc_fail_no_free.c

```diff
@@ -9,5 +9,7 @@ int main(void) {
   int *a = &x;
   int *b = malloc(4);
   int c = a[0];
-  return c;
+  // We have to use `b` otherwise the call to malloc will be removed.
```

Comment:
Do we have to make this change in other versions of this regression?

Reply:
No, I think only this benchmark was failing.

Reply:
Yes, but others passing just because code got optimized away?

Reply:
The corresponding benchmarks without errors seem to be well-defined so I don't think compiler optimizes anything.

Reply:
Ah, got it, makes sense.